packages/picasso.js/src/core/component/tween.js

```diff
@@ -37,9 +47,20 @@ export default function tween({ old, current }, { renderer }, config) {
       current.forEach((node, i) => {
         let id = trackBy(node, i);
         if (ids[id]) {
-          updated.ips.push(interpolateObject(ids[id], node));
+          if (node.type === 'path' && node.points && node.points.length > 0) {
+            const common = findCommonPointsFromTwoLines(ids[id], node);
+            updated.ips.push(
```

Comment:
we only interpolate the common points that should be updated from the old line node to the current line node. If we don't do this and interpolate all points, the animations would look very strange and far from what we expect.